engine/classes/Elgg/Http/ResponseFactory.php

```diff
@@ -144,7 +144,10 @@ public function send(Response $response) {
 						. (string) $this->response_sent);
 			}
 		} else {
-			$response->send();
+			if (elgg_trigger_before_event('send', 'http_response', $response)) {
```

Comment:
Another case for event params. As is handlers don't receive any contextual info that would help them to make a decision here.